tracer/src/Datadog.Trace.Tools.dd_dotnet/Datadog.Trace.Tools.dd_dotnet.csproj

```diff
@@ -64,11 +64,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
```

Comment:
As above. Also, this file is NativeAOT compiled, so all the dependencies are tree-shaken/trimmed anyway